querybook/webapp/components/AIAssistant/AICommandInput.tsx

```diff
@@ -139,6 +196,38 @@ export const AICommandInput: React.FC<AICommandInputProps> = forwardRef(
             </div>
         );
 
+        const loadTables = useCallback(
+            (
+                keywords: string,
+                callback: (
+                    options: Array<{ id: string; display: string }>
+                ) => void
+            ) => {
+                if (!keywords) {
+                    return;
+                }
+
+                SearchTableResource.searchConcise({
```

Comment:
I tried this out and noticed the first 2 letters doesn't return any responses, it would make this feature hard to discover for our users

Reply:
this should be some issue of my devapp index, it is supposed to start searching when people type at least one char.

Reply:
wait I also noticed we should be using SearchTableResource.suggest instead of search

Reply:
this piece of code was copied over from somewhere, didn't check them :( . let me revise them

Reply:
just realized that `.suggest` only works for prefix, I think `.searchConcise` might be more useful here, which can cover any word inside the table name.